kotsadm/web/src/components/modals/RestoreSnapshotModal.jsx

```diff
@@ -35,7 +35,9 @@ export default function RestoreSnapshotModal(props) {
             <p className="u-fontSize--normal u-fontWeight--normal u-color--dustyGray u-lineHeight--normal"> Restoring to this version will remove data and replace it with data from the restored version. During the restoration, your application will not be available and you will not be able to use the admin console. This action cannot be reversed. </p>
           </div>
           <div className="flex flex-column u-marginTop--20">
-            <p className="u-fontSize--normal u-fontWeight--bold u-color--tuna u-lineHeight--normal"> Type your application slug to continue </p>
+            <p className="u-fontSize--normal u-fontWeight--bold u-color--tuna u-lineHeight--normal"> Type your application slug to continue</p>
+
+            <p className="u-fontSize--normal u-fontWeight--normal u-color--dustyGray u-lineHeight--normal">We require you enter a matching app slug name as an additional confirmation you wish to proceed.</p>
```

Comment:
I'd like to make a copy change suggestion to better match how we've spoke about this in other similar scenarios like requiring app name to delete an app in vendor web. 

`To confirm that you want to delete this snapshot, please type it's slug in the input as it appears below.`

Reply:
After that change is made i'm good merging this. Thanks for removing Storybook until we can have a larger conversation about it's implementation.